projects/1.20/assets/productivebees/productivebees/patchouli_books/guide/zh_cn/entries/blocks/incubator.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "小蜜蜂孵化器",
+    "icon": "productivebees:incubator",
+    "category": "productivebees:blocks",
+    "pages": [
+        {
+            "type": "patchouli:spotlight",
+            "item": "productivebees:incubator",
+            "text": "等那些脆弱的小宝宝长大等到昏天黑地？培养箱能够加速这个过程。将幼蜂放入笼子中，再将其与一堆蜜蜂小食一起放入小蜜蜂孵化器里即可。"
+        },
+        {
+            "type": "patchouli:text",
+            "text": "孵化器还有另一个用途，它能向卵中注入特定的蜜蜂类型$(l:genetics/collecting#crystalline)基因$(/l)，以获得该类型蜜蜂的刷怪蛋。这里的蜜蜂类型基因需在遗传学设施中制作。$(br)制作刷怪蛋时只会使用类型基因，蜜蜂小食上其他所有的基因都会丢失。"
```

```suggestion
            "text": "孵化器还有另一个用途，它能向蛋中注入特定的蜜蜂类型$(l:genetics/collecting#crystalline)基因$(/l)，以获得该类型蜜蜂的刷怪蛋。这里的蜜蜂类型基因需在遗传学设施中制作。$(br)制作刷怪蛋时只会使用类型基因，蜜蜂小食上其他所有的基因都会丢失。"
```
经检验，此处接受的是蛋的标签，鸡蛋之类的都可以

---

其他位置不需要，但这个位置是因为拿来加工的物品是鸡蛋等，所以需要用蛋